docs/projects/fps/tutorial.md

```diff
@@ -657,11 +667,13 @@ The distributed game logic is now in place, and we can test if it is working cor
 ![A GIF showing the steps to enable the health bar UI]({{assetRoot}}assets/health-pickups-tutorial/health-bar-enable.gif)
 <%(/Expandable)%>
 
-<%(#Expandable title="2. Build your workers.")%>From the **SpatialOS** menu, select **Build UnityClient for local**.
+<%(#Expandable title="2. Build your workers.")%>Select **SpatialOS** > **Build For Local** > **UnityClient**.
 
 This is necessary because you have modified the code for the workers. If you are running your workers from within your Unity Editor a build is not necessary, however in a moment we will launch a built-out client-worker. Building the workers is therefore essential.<%(/Expandable)%>
 
-<%(#Expandable title="3. Launch a local deployment.")%>From the **SpatialOS** menu, select **Local launch**. This will open a terminal which will notify you when the deployment is up and running.
+<%(#Expandable title="3. Launch a local deployment.")%>From the **SpatialOS** menu, select **Local launch**. This will open a terminal which will notify you when the deployment is up and running.<br>
```

Comment:
```suggestion
<%(#Expandable title="3. Launch a local deployment.")%>From the **SpatialOS** menu, select **Local launch**. This opens a terminal which notifies you when the deployment is up and running.<br>
```

I know it's not your text - we try to avoid predictions about the future and make them statements of ever-present fact. :-)